dspace-api/src/main/java/org/dspace/workflow/Task.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.workflow;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.validation.constraints.NotNull;
+
+/**
+ * A workflow task.
+ */
+public class Task {
```

Comment:
Same here, improve JavaDocs please.  This isn't a Workflow "task" in the same manner as a ClaimedTask or PooledTask.  Instead, this seems to be defining a *Curation Task* that is mapped to a Workflow step.